### ISSUE TITLE
`quay.io/oauth2-proxy/oauth2-proxy:v7.5.0` doesn't work on `linux/arm64`

### ISSUE THREAD

**Issue by jsoref** [oauth2-proxy/oauth2-proxy#2231]
## Expected Behavior

pulled image should be for linux/arm64 and binaries should work

```sh
docker run -it --entrypoint sh quay.io/oauth2-proxy/oauth2-proxy:v7.4.0
/ $
```

## Current Behavior

```sh
% docker run -it --entrypoint sh quay.io/oauth2-proxy/oauth2-proxy:v7.5.0
exec /bin/sh: exec format error
```

## Possible Solution

Update the meta-image to include linux/arm64 (as v7.2.0, v7.3.0, and v7.4.0 have)

## Steps to Reproduce (for bugs)

1.  By a MacBook Pro M1 (or M2...)
2. Install [Rancher Desktop](https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.9.1)
3. Start Rancher Desktop (configure it to support `docker`)
4. Run `docker run -it --entrypoint sh quay.io/oauth2-proxy/oauth2-proxy:v7.5.0`

## Context

I was trying to debug a problem we experienced in an older version of oauth2-proxy to determine if it was fixed, but the out-of-the-box approach didn't work as the image label didn't include linux/arm64 (even though previous images did).

Note that it's possible to work around this on Macs as they have generally have emulation support, but it shouldn't be necessary...
```sh
% docker run --platform linux/amd64 -it --entrypoint sh quay.io/oauth2-proxy/oauth2-proxy:v7.5.0
Unable to find image 'quay.io/oauth2-proxy/oauth2-proxy:v7.5.0' locally
v7.5.0: Pulling from oauth2-proxy/oauth2-proxy
Digest: sha256:158d24014fb8fd4d4492c813edb97760cc1efd2738e7983b1f180b140f95314f
Status: Downloaded newer image for quay.io/oauth2-proxy/oauth2-proxy:v7.5.0
/ $
```
## Your Environment

- Version used: v7.5.0

**Comment by tuunit** [comment]
Hi @jsoref,

this issue was fixed with https://github.com/oauth2-proxy/oauth2-proxy/pull/2220 and we will release a patch version v7.5.1 in the upcoming days.

**Comment by kvanzuijlen** [comment]
Fixed with the release of 7.5.1
